application/models/user_access_token_test.go

```diff
@@ -145,36 +147,138 @@ func CreateUserAccessTokenFixtures(t *testing.T, user User, userOrgs UserOrganiz
 	return rawTokens
 }
 
+func CreateUserFixtures_GetOrg(ms *ModelSuite, t *testing.T) ([]Organization, Users, UserOrganizations) {
+	ResetTables(t, ms.DB)
+
+	unique := domain.GetUuid().String()
+
+	// Load Organization test fixtures
+	orgs := []Organization{
+		{
+			Name:       fmt.Sprintf("ACME-%s", unique),
+			Uuid:       domain.GetUuid(),
+			AuthType:   "saml2",
+			AuthConfig: "{}",
+		},
+		{
+			Name:       fmt.Sprintf("Starfleet Academy-%s", unique),
+			Uuid:       domain.GetUuid(),
+			AuthType:   "google",
+			AuthConfig: "{}",
+		},
+	}
+	for i := range orgs {
+		if err := ms.DB.Create(&orgs[i]); err != nil {
+			t.Errorf("error creating org %+v ...\n %v \n", orgs[i], err)
+			t.FailNow()
+		}
+	}
+
+	// Load User test fixtures
+	users := Users{
+		{
+			Email:     fmt.Sprintf("user1-%s@example.com", unique),
+			FirstName: "Existing",
+			LastName:  "User",
+			Nickname:  fmt.Sprintf("Existing User %s", unique),
+			Uuid:      domain.GetUuid(),
+		},
+		{
+			Email:     fmt.Sprintf("user2-%s@example.com", unique),
+			FirstName: "Another",
+			LastName:  "User",
+			Nickname:  fmt.Sprintf("Another User %s", unique),
+			Uuid:      domain.GetUuid(),
+		},
+	}
+	for i := range users {
+		if err := ms.DB.Create(&users[i]); err != nil {
+			t.Errorf("could not create test user %v ... %v", users[i], err)
+			t.FailNow()
+		}
+	}
+
+	// Load UserOrganization test fixtures
+	userOrgs := UserOrganizations{
+		{
+			OrganizationID: orgs[0].ID,
+			UserID:         users[0].ID,
+			AuthID:         users[0].Email,
+			AuthEmail:      users[0].Email,
+		},
+		{
+			OrganizationID: orgs[1].ID,
+			UserID:         users[1].ID,
+			AuthID:         users[1].Email,
+			AuthEmail:      users[1].Email,
+		},
+	}
+	for i := range userOrgs {
+		if err := ms.DB.Create(&userOrgs[i]); err != nil {
+			t.Errorf("could not create test user org ... %v. uo = %+v", err, userOrgs[i])
+			t.FailNow()
+		}
+	}
+
+	return orgs, users, userOrgs
```

Comment:
After I saw fillup use structs to return fixtures, I decided to use that idea going forward. If you prefer this, that's fine with me.